lib/client/backend_client.dart

```diff
@@ -11,8 +11,9 @@ class BackendClient {
   BackendClient({http.Client? httpClient})
       : httpClient = httpClient ?? http.Client();
 
-  Future<dynamic> getData({required String uri}) async {
-    var url = Uri.http(localhost, uri);
+  Future<dynamic> getData(
+      {required String uri, Map<String, dynamic>? queryParams}) async {
```

Comment:
Consider changing your Dart line length setting to 120 or 160 and reformatting all files. This looks like a line length of 80. It's up to you. Please resolve this comment.

Reply:
I changed the setting, but for some reason it isn't syncing, even when I reload vscode. Not sure why